### PR TITLE
fix(FEC-11896): Skip button aria label is incorrect

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEC-11896

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/skip/skip.js [hunk 1]
```diff
@@ -36,9 +36,11 @@ class Skip extends Component {
   render(): React$Element<any> | void {
     if (this.props.playerSize !== PLAYER_SIZE.TINY && !this.props.loading) {
       return (
-        <div tabIndex="0" aria-label={this.props.label} className={skipStyle.btnSkip} onClick={this.props.onClick}>
-          <Text id={this.props.label} />
-        </div>
+        <Localizer>
```
Comment (yairans) [inline src/components/skip/skip.js:39]: Localizer is part of the fix? how?

Comment (JonathanTGold) [reply src/components/skip/skip.js:39]: https://github.com/synacor/preact-i18n#localizer

Comment (yairans) [reply src/components/skip/skip.js:39]: I know what is Localizer. just asked if it's relevant to the aria label only or it should be added also before, for the label itself?

Comment (JonathanTGold) [reply src/components/skip/skip.js:39]: relevant to the aria label only
<Localizer /> is a Compositional Component. It "renders" out any <Text /> values **in its child's props.**
a quote from the link

**Review by yairans — APPROVED** [review]

**Pull request merged**